rts/Rendering/Units/UnitDrawer.cpp

```diff
@@ -547,18 +548,25 @@ void CUnitDrawerLegacy::DrawUnitIconsScreen() const
 			const float x1 = (pos.x + offset) / globalRendering->viewSizeX;
 			const float y1 = (pos.y - offset) / globalRendering->viewSizeY;
 
-			if (x1 < 0 && x0 > 1 && y0 < 0 && y1 > 1)
+			if (x1 < 0 || x0 > 1 || y0 < 0 || y1 > 1) // This was previously &&, if any of these are true, then the icon is not drawn.
```

Comment:
"this was previously incorrect behaviour" doesn't sound like a helpful comment
```suggestion
			if (x1 < 0 || x0 > 1 || y0 < 0 || y1 > 1)
```

Reply:
Indeed, that is not helpful. Should I separate commits so that gitlens will show why the line was changed?

Reply:
Nah, `x < 0 && x > 1` is obviously incorrect so doesn't really need an explanation. In theory there are two separate changes here (fix the culling logic, and avoid a texture switch) but both are trivial enough that I don't think it's worth the bother.